api/debug/debug_test.go

```diff
@@ -156,13 +157,22 @@ func TestStorageRangeMaxResult(t *testing.T) {
 	assert.Equal(t, 10, len(storageRangeRes.Storage))
 }
 
+func testTraceClauseWithEmptyTracerName(t *testing.T) {
```

Comment:
do we have any test for invalid tracer name ?

Reply:
Added !